addons/mail/static/src/components/discuss_sidebar_mailbox/discuss_sidebar_mailbox.scss

```diff
@@ -0,0 +1,57 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_DiscussSidebarMailBox {
```

Comment:
`DiscussSidebarMailBox` => `DiscussSidebarMailbox` (rename all classnames from "MailBox" to "Mailbox")